govuk_sidekiq.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir.glob("test/**/*")
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.2"
+  spec.required_ruby_version = ">= 2.6"
 
   spec.add_dependency "sidekiq", ">= 5", "< 6"
```

Comment:
I wonder if we can loosen this at the same time, since 6 has been out since Nov 2019. My brief scan of the change log didn't seem to show any blocking issues.

Reply:
Yes, the [upgrade guide](https://github.com/mperham/sidekiq/blob/master/6.0-Upgrade.md) looks reasonable to me, I can't see any blockers.  I'll include that.

Reply:
Ew, sidekiq-logging-json hasn't been updated in over a year...

Reply:
I'll leave that for a later PR I think